src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static final String SHORT_LOGIN = "short";
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String SHORT_PASSWORD = "short";
+    private static final String VALID_PASSWORD = "validPassword";
+    private static final Integer VALID_AGE = 20;
+    private static final Integer UNDERAGE = 17;
+    private static final String EXISTING_LOGIN = "existingLogin";
+
+    private static RegistrationService registrationService;
+    private static StorageDaoImpl storageDao;
+
+    @BeforeAll
+    public static void setUp() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
```

Comment:
Consider whether `@BeforeAll` is the best choice here. Since `RegistrationServiceImpl` does not seem to have any state that needs to be shared between tests, `@BeforeEach` might be more appropriate to ensure that each test has a fresh instance of the service, which prevents tests from affecting each other.